built_value_generator/lib/src/value_source_class.dart

```diff
@@ -109,9 +116,11 @@ abstract class ValueSourceClass
   }
 
   @memoized
-  BuiltList<String> get genericParameters =>
-      new BuiltList<String>(element.typeParameters
-          .map((element) => element.computeNode().toString()));
+  BuiltList<String> get genericParameters => new BuiltList<String>(
+      element.typeParameters.map((element) => element.name));
+//  BuiltList<String> get genericParameters =>
+//      new BuiltList<String>(element.typeParameters.map((element) =>
+//          parsedLibrary.getElementDeclaration(element).node.toSource()));
```

Comment:
Ah, here is another case like below.
Remove commented out previous implementation?

Reply:
Done, thanks.